apps/trucking-kit/src/scss/theme.scss

```diff
@@ -3,6 +3,29 @@
 @use './typography';
 @use '../../../../libs/shared/ui/src/index' as shared-ui;
 
+@function mat-set-background($theme, $backgroundColor) {
```

Comment:
Are we set up to use M3 theming? If so, I wonder if it's necessary for us to use this function to overwrite the background, or if we could just re-declare the background css variable in root?

Reply:
I wish, the m3 theming is still limited, last I checked, there was no way to customize it, just some preset colors